org-code-javabuilder/lib/src/main/java/org/code/javabuilder/AWSContentManager.java

```diff
@@ -43,8 +43,16 @@ public AWSContentManager(
       String bucketName,
       String javabuilderSessionId,
       String contentBucketUrl,
-      Context context) {
-    this(s3Client, bucketName, javabuilderSessionId, contentBucketUrl, context, null);
+      Context context)
+      throws InternalServerError {
+    this.bucketName = bucketName;
+    this.s3Client = s3Client;
+    this.javabuilderSessionId = javabuilderSessionId;
+    this.contentBucketUrl = contentBucketUrl;
+    this.context = context;
+    this.projectData = this.loadProjectData();
```

Comment:
Is the other constructor just for tests?

Reply:
Yep! I couldn't call the other constructor with `this(...)` since I needed to initialize project data with `this.loadProjectData()`